applications/MultilevelMonteCarloApplication/python_scripts/pure_diffusion_solver.py

```diff
@@ -0,0 +1,230 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
this solver should not be committed. You should use the one in convection diffusion